components/automate-ui/src/app/pages/+compliance/+profile/+profile-details/profile-details.component.html

```diff
@@ -35,23 +35,25 @@
       <chef-page-header>
         <chef-heading>{{ profile.title }}</chef-heading>
         <chef-subheading>{{ profile.summary }}</chef-subheading>
-        <div slot="header-buttons">
-          <ng-container *ngIf="!isInstalled">
-            <chef-button secondary caution id="install-btn" (click)="installProfile(profile)">
-              <chef-icon>get_app</chef-icon>
-              <span>Get</span>
+        <div class="header-buttons-container">
```

Comment:
Added a container <div> so that the actual buttons container could be targeted only in profile pages.  The `header-buttons` <div> is rendered as a slot and I'm unable to target locally without this container.